tmuxp/workspacebuilder.py

```diff
@@ -264,9 +269,6 @@ def build(self, session=None, append=False):
                 assert isinstance(p, Pane)
                 p = p
 
-                if "layout" in wconf:
-                    w.select_layout(wconf["layout"])
-
```

Comment:
@nvasilas Do you recall the rationale behind the removing / adding of the `.select_layout`'s here, [here](https://github.com/tmux-python/tmuxp/pull/793/files#diff-9316655a97cbe443559ee45c1b1fa26bfa86e314210653806ce215702a956931R286), and [here](https://github.com/tmux-python/tmuxp/pull/793/files#diff-9316655a97cbe443559ee45c1b1fa26bfa86e314210653806ce215702a956931L424)?

Reply:
@tony If I recall correctly without this change the layout wasn't updated properly. It seems like I broke some things with this PR. I will investigate and I'll try to provide a fix.

Reply:
@nvasilas Thank you! Let's see how it goes!